src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -0,0 +1,128 @@
+                        <li>Use the up and down arrow keys to traverse the list</li>
+                        <hr className="tw-my-2 tw-border-black-20 tw-bg-black-20" />
+                        <li>Use the left and right arrow keys to switch focus to the prev/next element</li>
+                    </ul>
+                ),
+                alt: "Six",
+            },
+            {
+                id: "1",
+                type: "item",
+                content: (
+                    <div>
+                        <p>Use the prevKey and nextKey props to determine the position of items in the list</p>
+                    </div>
+                ),
+                alt: "Four",
+            },
+            {
+                id: "2",
+                type: "item",
+                content: (
+                    <>
+                        <Checkbox value={CheckboxState.Checked} label="List Item With Checkbox" />
+                    </>
+                ),
+                alt: "Checkbox Item",
+            },
+            {
+                id: "3",
+                type: "item",
+                content: <Button>List Item With Button</Button>,
+                alt: "List Item With Checkbox",
+            },
+            {
+                id: "4",
+                type: "item",
+                content: (
+                    <div>
+                        <p>List Item With multiple focusable elements</p>
+                        <div className="tw-flex tw-justify-around tw-mt-2">
+                            <Checkbox value={CheckboxState.Unchecked} label="First Input" />
+                            <Checkbox value={CheckboxState.Unchecked} label="Second Input" />
+                            <Checkbox value={CheckboxState.Unchecked} label="Third Input" />
+                            <Checkbox value={CheckboxState.Unchecked} label="Fourth Input" />
+                        </div>
+                    </div>
+                ),
+                alt: "text",
+            },
+            { id: "5", type: "item", content: "Five", alt: "Five" },
+        ],
+    });
+
+    const onMove = (selectedGridItemKeys: React.Key[], gridItemLocation: ItemDropTarget) =>
+        gridItemLocation.dropPosition === "before"
+            ? list.moveBefore(gridItemLocation.key, selectedGridItemKeys)
+            : list.moveAfter(gridItemLocation.key, selectedGridItemKeys);
+
+    return (
+        <div className="tw-m-auto  tw-w-[600px]">
+            <OrderableList
+                items={list.items}
+                onMove={onMove}
+                dragDisabled={dragDisabled}
+                renderContent={renderContent}
+            />
+            <div className="tw-flex tw-justify-center tw-mt-3">
+                <Button onClick={() => setDragDisabled((prev) => !prev)}>
+                    {`${dragDisabled ? "Enable" : "Disable"} Drag`}
+                </Button>
+            </div>
+        </div>
+    );
+};
+
+export default {
```

Comment:
Just a preference thing but I'd suggest moving this part to the top of the file like in all other stories

Reply:
done